styles.css

```diff
@@ -0,0 +1,736 @@
+@import url("http://fonts.cdnfonts.com/css/cocogoose");
+@import url("http://fonts.cdnfonts.com/css/lato");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Lato, sans-serif;
+}
+
+h1,
+h2,
+h3 {
+  font-family: COCOGOOSE, sans-serif;
+}
+
+h1 {
+  color: #ec5242;
+  text-transform: uppercase;
+  font-size: 30px;
+  font-weight: normal;
+}
```

Comment:
- [ ] Let's make the header of the pages to be bold just like the design template. You can play around with the `font-size` and `font-weight` till you get the style that looks closely to the design template. Ensure the `about` page header is affected by these changes too 😉.

Your Implementation | Cindy's Design
:---:|:---:
![](https://i.imgur.com/goDPzcQ.png) | ![](https://i.imgur.com/An3UVgO.png)

Reply:
This is very strange. In my local version, the letters were showing bold. I was looking at the preview from VS Code. In my screenshot attached to the readme, you can see that the letters show bolder than the ones deployed on github, but when I opened the deployed version, it is really not bold.
Thank you for noticing, I will fix that immediately.